src/http/sr-modules-operators-keys/sr-modules-operators-keys.controller.ts

```diff
@@ -32,7 +39,36 @@ export class SRModulesOperatorsKeysController {
     description: 'Staking router module_id or contract address.',
   })
   @Get(':module_id/operators/keys')
-  getOperatorsKeys(@Param('module_id') moduleId: ModuleId, @Query() filters: KeyQuery) {
-    return this.srModulesOperatorsKeys.get(moduleId, filters);
+  async getOperatorsKeys(
+    @Param('module_id') moduleId: ModuleId,
+    @Query() filters: KeyQuery,
+    @Res() reply?: FastifyReply,
+  ) {
+    await this.entityManager.transactional(
+      async () => {
+        const { operators, keysGenerator, module, meta } = await this.srModulesOperatorsKeys.get(moduleId, filters);
+
+        const jsonStream = JSONStream.stringify(
+          '{ "meta": ' +
+            JSON.stringify(meta) +
+            ', "data": { "module": ' +
+            JSON.stringify(module) +
+            ', "operators": ' +
+            JSON.stringify(operators) +
```

Comment:
if we assume that there will be as many operators as not fit in VM memory, `stringify` method will fail.